From 2dee9cd1ae172fac456dc486f9d38744b33ff006 Mon Sep 17 00:00:00 2001
From: Zebediah Figura <z.figura12@gmail.com>
Date: Mon, 18 May 2020 14:20:30 -0500
Subject: [PATCH 10/17] HACK: steam: wine.inf: Associate the steam protocol
 with steam.exe.

For 2K Launcher.

CW-Bug-Id: 18912
(cherry picked from commit eeb10d71bea438089b62906f0aaa40be1a05dc97)
---
 loader/wine.inf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 0986e02e2b6..0bd23fb42f1 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -308,6 +308,7 @@ HKCR,ftp\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,http\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,https\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,mailto\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
+HKCR,steam\shell\open\command,,,"""%16426%\Steam\Steam.exe"" -- ""%1"""
 
 [ContentIndex]
 HKLM,System\CurrentControlSet\Control\ContentIndex\Language\Neutral,"WBreakerClass",,"{369647e0-17b0-11ce-9950-00aa004bbb1f}"
-- 
2.55.0

